### DISCUSSION TITLE
[FEATURE] 优化面板 高级功能-WAF-白名单 功能的建议

### DISCUSSION THREAD

**Discussion by yuanmoe** [1Panel-dev/1Panel#5967] [Ideas]
### 1Panel 版本

专业版 v1.10.13-lts

### 请描述您的需求或者改进建议

管理员在做测试的过程中经常被 WAF 的各种规则拦截。

### 请描述你建议的实现方案

自动过白成功登录面板的 IP （过白一段时间？直到退出面板登录？反正总不能是永久吧，具体我也没想好）

**Comment by yuanmoe** [comment]
尤其是访问频率限制，多刷新几次我 IP 就被 BAN 了，然后我就得拿起手机开热点，登录面板，设置自己为白名单

**Comment by zhengkunwang223** [comment]
> 尤其是访问频率限制，多刷新几次我 IP 就被 BAN 了，然后我就得拿起手机开热点，登录面板，设置自己为白名单

可以使用 URL 模式，降低误封几率
